packages/web-twig/DEPRECATIONS-v3.md

```diff
@@ -4,7 +4,166 @@ Following deprecations will be removed in version 3 of the _spirit-web-twig_ pac
 
 > Please follow these steps to safely upgrade your design system to Spirit Design System v3 components.
```

Comment:
As mention in the `web` PR, I think only package version should be here. The common version is rather a marketing thing and I'm hesitating to legalize it by using it in our code…

Reply:
it is already there. We used this versioning in our migration guides in the summer.

Reply:
I will leave it like it is. Feel free to open a topic on our Thursday meetings